src/Request/RequestInterface.php

```diff
@@ -8,7 +8,7 @@
 
 interface RequestInterface
 {
-    public function getMethod(): RequestEnum;
+    public function getFunction(): RequestEnum;
```

Comment:
Wouldn't `endpoint` be better?

Reply:
I wanted to stick to SoapClient terms

Reply:
Still `endpoint` would be more generic and we don't need to keep nomenclature consistent with underlying solution. Anyway I won't be insisting here :)